domain/consensus/processes/consensusstatemanager/pick_virtual_parents.go

```diff
@@ -46,7 +45,12 @@ func (csm *consensusStateManager) pickVirtualParents(tips []*externalapi.DomainH
 			end--
 		}
 	}
+	// Limit to 30 candidates, that way we don't go over thousands of tips when the network isn't healthy.
+	if len(candidates) > int(csm.maxBlockParents)*3 {
```

Comment:
Please extract maxBlockParent*3 to a constant (in constants.go) and add a comment over it explaining why that number was chosen.

Reply:
Well there's no reason it was chosen other than my whims